README.md

```diff
@@ -5,6 +5,7 @@
 
 <!-- TABLE OF CONTENTS -->
 
+
 # 📗 Table of Contents
```

Comment:
- [ ] Please make the foreign key between the posts table and the users table as `author_id` because now `Active Record` will create it as `user_id` because of naming conventions.

https://github.com/Nowembabazi/BLOG-APP/blob/3624e2ab1c31e35b929d8dba833f38158c46058b/db/migrate/20230207102605_add_foreign_key_to_posts.rb#L2-L4

[Hint] You can edit the migration to be like that:
```ruby
add_reference :posts, :author, null: false, foreign_key: { to_table: :users }
```



- [ ] Nice job adding references between the posts table and comments table, Please also add references between the users table and the comments table as `author_id` as I showed above.

https://github.com/Nowembabazi/BLOG-APP/blob/3624e2ab1c31e35b929d8dba833f38158c46058b/db/migrate/20230207102149_add_foreign_key_to_comments.rb#L2-L4



- [ ] Same here, Please add references between the users table and the likes table as `author_id`.

https://github.com/Nowembabazi/BLOG-APP/blob/3624e2ab1c31e35b929d8dba833f38158c46058b/db/migrate/20230207102450_add_foreign_key_to_likes.rb#L2-L4